src/sdk/pynni/nni/compression/torch/pruning/one_shot.py

```diff
@@ -134,9 +134,10 @@ def __init__(self, model, config_list, pruning_algorithm, optimizer=None, **algo
 
     def validate_config(self, model, config_list):
         schema = CompressorSchema([{
-            'sparsity': And(float, lambda n: 0 < n < 1),
-            'op_types': ['Conv2d'],
-            Optional('op_names'): [str]
+            Optional('sparsity'): And(float, lambda n: 0 < n < 1),
```

Comment:
sparsity becomes optional?

Reply:
If  `exclude` is specified, sparsity is not necessary.

Reply:
got it